spec/services/bolognese/writers/hyrax_work_writer_behavior_spec.rb

```diff
@@ -98,4 +98,40 @@
       end
     end
   end
+
+  describe "#cleanup_author" do
```

Comment:
I think we might be missing a test where one or both of the author's names are blank, but not nil. Can that happen?